### ISSUE TITLE
Long targets in Jupyter Notebook, Anaconda Navigator and Spyder cause improper execution from menu items

### ISSUE THREAD

**Issue by dbw44** [ContinuumIO/anaconda-issues#9356]
I have not been able to execute Jupyter Notebook from the Windows 10 menu, and when I look at the Target field in the properties of the menu item, I see that the directory names are very long and as a result, the final directory in the Target is truncated without a quote mark; also, forward slashes are used instead of back slashes.  This is also true of the items for Anaconda Navigator and Spyder, although these can be executed correctly for some reason.    Please use short filenames in these targets to prevent them from being truncated, so that they will execute properly.

##### `conda info`

```

```

##### `conda list --show-channel-urls`

```

```

**Comment by goanpeca** [comment]
Please provide the requested information on the issue template

**Comment by mingwandroid** [comment]
> I see that the directory names are very long and as a result, the final directory in the Target is truncated without a quote mark

You are incorrect. The Windows control in this dialog that shows the shortcut is limited to 256 characters, but the internal storage in the registry allows many more (32k AFAIR). You should not edit it through this control though. That would break it.

> also, forward slashes are used instead of back slashes

How is this a problem? The WIN32 API is fine with either type of slash.

Your issue is something different (probably DLL-hell). However if you did try to edit these shortcuts then you will have broken them. If you feel like you should edit them (though we don't support that) then you need to use the Registry Editor to do so.
